apollo-api/src/commonMain/kotlin/com/apollographql/apollo3/api/Adapters.kt

```diff
@@ -214,9 +223,13 @@ class ObjectAdapter<T>(
   }
 }
 
+@JvmName("-nullable")
 fun <T : Any> Adapter<T>.nullable() = NullableAdapter(this)
```

Comment:
This hides the `nullable` method from Java. Doing `new NullableAdapter(wrappedAdapter())` feels more idiomatic and better have only one way to do it?